Add GPU tested compatibility in documentation (#300)
cncastillo/KomaMRI.jl: README.md

         eprint = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/mrm.29635},
}
```

> ## Nvidia GPU tested compatibility

Reviewer: ```suggestion
## Tested compatibility
```

> We automatically test KomaMRI.jl using GitHub Actions. So far, we have implemented automatic tests for the most popular operating systems on CPUs, which is precisely what the [![][gh-actions-img1]][gh-actions-url] badge reflects. Generating automatic tests for GPUs is still a work in progress; however, we ensure that everything works flawlessly on our own machines. Here is a summary table for local GPU tests:

Reviewer: ```suggestion
We automatically test KomaMRICore's CPU multi-threading support on the most popular operating systems. Nevertheless, for GPU support the process is more manual (until #147). Here is a summary of our automatic CPU tests and local GPU tests for multiple versions of Julia:
```

| Julia (OS)                            | CPU                                       | GPU (Nvidia) |
|:--------------------------------------|:----------------------------------------------:|:------------------:|
| Julia 1.9 (Windows)  | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
| Julia 1.9 (Linux)    | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
| Julia 1.9  (Mac)      | [![][gh-actions-img]][gh-actions-url]          | ➖                |
| Julia 1.10 (Windows)           | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
| Julia 1.10 (Linux)             | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
| Julia 1.10 (Mac)               | [![][gh-actions-img]][gh-actions-url]          | ➖                |
| Julia 1.11 (Windows)          | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ❌                 |
| Julia 1.11 (Linux)            | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ❌                 |

> | Julia 1.11 (Mac)              | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ➖                 |

Reviewer: ```suggestion
| Julia 1.11 (Mac)              | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ➖                 |
```

if you see any problem with this information, please let us know in the form of a GitHub issue.

> | Julia / OS                            | CPU (CI)                                       | Nvidia GPU (local) |
> |:--------------------------------------|:----------------------------------------------:|:------------------:|
> | Julia 1.9 - oldest supported / Windows  | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
> | Julia 1.9 - oldest supported / Linux    | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
> | Julia 1.9 - oldest supported / Mac      | [![][gh-actions-img]][gh-actions-url]          | ➖                |
> | Julia 1.10 - latest / Windows           | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
> | Julia 1.10 - latest / Linux             | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
> | Julia 1.10 - latest / Mac               | [![][gh-actions-img]][gh-actions-url]          | ➖                |
> | Julia 1.11 - nightly / Windows          | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ❌                 |
> | Julia 1.11 - nightly / Linux            | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ❌                 |
> | Julia 1.11 - nightly / Mac              | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ➖                 |

Reviewer: ```suggestion
| Julia (OS)                            | CPU                                       | GPU (Nvidia) |
|:--------------------------------------|:----------------------------------------------:|:------------------:|
| Julia 1.9 (Windows)  | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
| Julia 1.9 (Linux)    | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
| Julia 1.9  (Mac)      | [![][gh-actions-img]][gh-actions-url]          | ➖                |
| Julia 1.10 (Windows)           | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
| Julia 1.10 (Linux)             | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
| Julia 1.10 (Mac)               | [![][gh-actions-img]][gh-actions-url]          | ➖                |
| Julia 1.11 (Windows)          | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ❌                 |
| Julia 1.11 (Linux)            | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ❌                 |
| Julia 1.11 (Mac)              | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ➖                 |
```
